sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/taskdata/EnvConstants.java

```diff
@@ -25,9 +25,10 @@ private EnvConstants() {
      *  under this domain. For example, <VIP_NAME>.<FRAMEWORK_VIP_HOST> => vip hostname. */
     public static final String FRAMEWORK_VIP_HOST_TASKENV = "FRAMEWORK_VIP_HOST";
 
-    /** Provides the hostname of the Scheduler's own API. Pods within the service may access scheduler-hosted APIs under
-     *  this hostname. For example, <SCHEDULER_API_HOSTNAME>/v1/state/files/notavirus.exe.txt */
+    /** Provides the hostname/port of the Scheduler's own API. Pods within the service may access scheduler-hosted APIs
+     *  under this hostname. For example, &lt;hostname>:&lt;port>/v1/state/files/notavirus.exe.txt */
```

Comment:
I don't know what happened to these angle brackets

Reply:
this fixes rendering in javadoc html